source/NVDAObjects/window/winword.py

```diff
@@ -1150,6 +1180,13 @@ def script_previousColumn(self,gesture):
 
 class WordDocument(Window):
 
+	def _get_screenUpdatingDisabler(self):

```

Comment:
May be explicitly state that this should be cached?

```suggestion
	_cache_screenUpdatingDisabler = True
	def _get_screenUpdatingDisabler(self):
```